tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/BrokerContainer.java

```diff
@@ -29,4 +31,29 @@ public BrokerContainer(String clusterName, String hostName) {
         super(
             clusterName, hostName, hostName, "bin/run-broker.sh", BROKER_PORT, INVALID_PORT);
     }
+
+    public void stopBroker() {
+        try {
+            this.execCmd("supervisorctl", "stop", "broker");
+        } catch (Exception ex){
+
+        }
+    }
+
+    public void startBroker() {
+        try {
+            this.execCmd("supervisorctl", "start", "broker");
+        } catch (Exception ex){
+        }
+    }
+
+    public void updateConf(String configFile, String key, String value) {
```

Comment:
any configuration updates should just be done by specifying the configuration settings as docker environment variables. 

for example: https://github.com/apache/incubator-pulsar/blob/master/tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java#L159